rs/embedders/src/wasm_utils/instrumentation.rs

```diff
@@ -1380,7 +1380,7 @@ enum Scope {
 #[derive(Copy, Clone, Debug, PartialEq)]
 enum InjectionPointCostDetail {
     StaticCost { scope: Scope, cost: u64 },
-    DynamicCost,
+    DynamicCost { extend64_bit: bool },
```

Comment:
One of the recommended practices is to avoid boolean flags because the reader can get confused about the meaning of `true` and `false` when at the call site:
```
DynamicCost {true}
DynamicCost {false}
```

It would be more readable to have an enum with descriptive names. For example:
```
DynamicCost { CostOperandOnStack::X32Bit }
DynamicCost { CostOperandOnStack::X64Bit }
```
Then the reader will immediately see that `DynamicCost` depends on the operand on stack and wants to know its size.

Reply:
Thank you, fixed now.